ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoInputHelper.java

```diff
@@ -116,6 +116,8 @@ public static void synchronize(Crac crac, Network network) {
         });
         noValidAction.forEach(networkAction -> crac.removeNetworkAction(networkAction.getId()));
 
+        // remove onContingency Remedial Action
```

Comment:
is this a TODO?

Reply:
yes :), this is the initial purpose of the PR, but in the end I decided to divide it in another PR for them to be easier to read. You are right i will put a TODO its not clear